src/components/BulkAssignment/BulkAssignmentModal.jsx

```diff
@@ -317,4 +317,4 @@ BulkAssignmentModal.defaultProps = {
 
 BulkAssignmentModal.displayName = 'BulkAssignmentModal';
 
-export default connectModal(BulkAssignmentModal);
+export default connectModal(BulkAssignmentModal);
```

Comment:
```suggestion
export default connectModal(BulkAssignmentModal);

```
^ Prettier is throwing a fit about this line

Reply:
Fixed